src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,16 @@
 import React from 'react';
 import './MoviesList.scss';
+import { MovieCard } from '../MovieCard';
+import { Movie } from '../../types/Movie';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  movies: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = ({ movies }) => (
+  <div className="movies">
+    {movies.map((movie) => (
+      <MovieCard movie={movie} key={movie.imdbUrl} />
```

Comment:
More appropriate is to use id ( `imdbId` in this case)
```suggestion
      <MovieCard movie={movie} key={movie.imdbId} />
```

Reply:
Fixed